.circleci/config.yml

```diff
@@ -106,7 +106,7 @@ references:
           && ./configure && make -j$(nproc) && sudo make install && sudo ldconfig
 
   tag_regex: &tag_regex /^v.*$/
-  stable_tag_regex: &stable_tag_regex /^v4\..*$/
+  master_branch: &master_branch lima
```

Comment:
Restoration of this tracked as https://www.pivotaltracker.com/story/show/167894277